pkg/apiserver/statement/statement.go

```diff
@@ -210,3 +228,118 @@ func (s *Service) getPlanDetailHandler(c *gin.Context) {
 	}
 	c.JSON(http.StatusOK, result)
 }
+
+// @Router /statements/download/acquire_token [get]
```

Comment:
This operation leaves something on the disk. Should it be PUT or POST instead?

Reply:
Done.